src/components/register-in-contest/register-project.jsx

```diff
@@ -64,16 +65,32 @@ export default React.createClass({
                    required />,
             <p className="alert alert-warning">
               Sursele proiectului trebuie să fie încărcate și accesibile pe
-              GitHub. În cazul în care întâmpini probleme poți găsi mai multe
-              detalii aici.
+              GitHub. În cazul în care întâmpini probleme poți găsi mai multe detalii&nbsp;
+              <a target="_blank" href="http://blog.infoeducatie.ro/tutorial/2015/04/14/github-101.html">aici</a>.
             </p>
         ]);
       } else {
-        wantsOpenSource = <Input type="input"
-                                 placeholder="Îmi este mult prea frică că îmi va fura un om rău codul"
-                                 valueLink={this.deepLinkState(["project", "closed_source_reason"])}
-                                 label="Care este motivul pentru care dorești ca proiectul tău să nu fie public (open-source) ? *"
-                                 required />;
+        wantsOpenSource = ([
+            <Input type="input"
+                   placeholder="Îmi este mult prea frică că îmi va fura un om rău codul"
```

Comment:
Eu l-aș lăsa. Nu cred că e la mișto / bâză etc. Doar îi încurajăm, într-un mod comic, să își facă munca open-source. Când terminăm articolul cu `De ce este open-source important?` le băgăm și un link.